services/buildkite/buildkite.tester.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const ServiceTester = require('../service-tester');
+const t = new ServiceTester({ id: 'buildkite', title: 'Buildkite Builds' });
+const { invalidJSON } = require('../response-fixtures');
+module.exports = t;
+
+t.create('buildkite invalid pipeline')
+  .get('/unknown-identifier/unknown-branch.json')
+  .expectJSON({ name: 'build', value: 'not found' });
+
+t.create('buildkite valid pipeline')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489/master.json')
+  .expectJSON({ name: 'build', value: 'passing' });
+
+t.create('buildkite valid pipeline skipping branch')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489.json')
+  .expectJSON({ name: 'build', value: 'passing' });
+
+t.create('buildkite unknown branch')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489/unknown-branch.json')
+  .expectJSON({ name: 'build', value: 'unknown' });
+
+t.create('buildkite connection error')
+  .get('/_.json')
+  .networkOff()
+  .expectJSON({ name: 'build', value: 'inaccessible' });
+
+t.create('buildkite unexpected response')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489.json?branch=master')
```

Comment:
I think you may be able to leave out the `?branch=master` from this part of the request.

Reply:
How come? What’s makes it different from the one below?

Reply:
this 1st is the address you would enter to get the badges image.
eg:
```
https://img.shields.io/buildkite/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489.json?branch=master
```
would only need to be
```
https://img.shields.io/buildkite/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489.json
```
or
```
https://img.shields.io/buildkite/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489/master.json
```
--------------
the 2nd is the request we want to intercept to the buildkite server:
```
https://badge.buildkite.com/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489.json?branch=master
```

Reply:
Ooohh I see. Thanks!👌